packages/core-mobile/app/hooks/earn/useDefaultFeeState.ts

```diff
@@ -0,0 +1,31 @@
+import { pvm } from '@avalabs/avalanchejs'
+import { useQuery, UseQueryResult } from '@tanstack/react-query'
+import { useSelector } from 'react-redux'
+import NetworkService from 'services/network/NetworkService'
+import { selectActiveNetwork } from 'store/network'
+import { selectIsDeveloperMode } from 'store/settings/advanced'
+import { isDevnet } from 'utils/isDevnet'
+
+export const useDefaultFeeState = (): UseQueryResult<
+  pvm.FeeState | undefined,
+  Error
+> => {
+  const isDeveloperMode = useSelector(selectIsDeveloperMode)
+  const network = useSelector(selectActiveNetwork)
+  const devnet = isDevnet(network)
+
+  return useQuery({
+    retry: false,
+    queryKey: ['defaultFeeState', isDeveloperMode, devnet],
+    queryFn: async () => {
```

Comment:
should we add interval fetching too? every 30 seconds?

Reply:
updated to 1min